api/HMDVRDevice.json

```diff
@@ -0,0 +1,214 @@
+{
+  "api": {
+    "HMDVRDevice": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HMDVRDevice",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "The support in Chrome is currently experimental. To find information on Chrome's WebVR implementation status including supporting builds, check out <a href='http://blog.tojicode.com/2014/07/bringing-vr-to-chrome.html'>Bringing VR to Chrome</a> by Brandon Jones."
```

Comment:
It's in an [origin trial](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/available-trials.md) starting in 62. Also, it's in [`chrome_android` not `chrome`](https://www.chromestatus.com/feature/4532810371039232). `webview_android` and `chrome` should both say `false`.